fastapi_cache/decorator.py

```diff
@@ -81,7 +82,23 @@ def _uncacheable(request: Optional[Request]) -> bool:
         return False
     if request.method != "GET":
         return True
-    return request.headers.get("Cache-Control") == "no-store"
+    return False
+
+def _extract_cache_control_headers(request: Optional[Request]) -> Set[str]:
+    """Extracts Cache-Control header
+    1. Convert all header to lowercase to make it case insensitive
+    2. Split on comma (,)
+    3. Strip whitespaces
+    4. convert to all lower case
+
+    returns an empty set if header not set
+    """
+    if request is not None:
+        headers = {header_key.lower(): header_val for header_key, header_val in request.headers.items()}
```

Comment:
I think we can omit lowecasing since FastAPI uses starlette where headers are case-insensitive https://fastapi.tiangolo.com/tutorial/header-params/?h=insensitive#automatic-conversion